package.json

```diff
@@ -25,6 +25,7 @@
   "dependencies": {
     "d3": "^3.5.17",
     "lodash": "^4.13.1",
+    "mark.js": "github:julmot/mark.js#649fdcd",
```

Comment:
But using a commit hash locks down the dependency. I would think you'll be releasing future version of the library and updating the hash here every time you do so is counter intuitive.